src/styles.css

```diff
@@ -0,0 +1,284 @@
+    padding: 0;
+    box-sizing: border-box;
+    scrollbar-width: none;
+}
+
+::-webkit-scrollbar {
+    width: 0;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    color: var(--color-text);
+    background-color: var(--color-primary-black);
+    /* overflow: hidden; */
+    /* scrollbar-width: none; */
+}
+
+.container-app {
+    display: flex;
+    height: 90vh;
+    width: 100%; 
+}
+
+/* LOGIN */
+.login {
+    height: 100vh;
+    display: grid;
+    place-items: center;
+}
+
+.container-login {
+    background-color: var(--color-secondary-black);
+    padding: 2rem;
+    margin: 0 1rem;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container-login button {
+    border: none;
+    border-radius: 3px;
+    padding: 0.5rem;
+    margin-bottom: 1rem;
+}
+
+.container-login button:focus {
+    outline: none;
+    cursor: pointer;
+}
+
+/* HEADER */
+.container-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 3rem;
+    background-color: var(--color-secondary-black);   
+    height: 10vh;
+    margin: 0;
+}
+
+.header-hamburger {
+    display: none;
+}
+
+.header-brand {
+    display: block;
+}
+
+.header-form {
+    flex: 0.6;
+    display: flex;
+    padding: 0.1rem;
+    margin: 0.1rem;
+    border-radius: 2px;
+    border: 1px solid var(--color-border);
+    background-color: var(--color-secondary-black);
+}
+
+.header-form input {
+    width: 100%;
+    border: none;
+    background: transparent;
+    font-size: 0.9em;
+    font-weight: bold;
+    padding: 0.8rem 0.5rem;
+    color: var(--color-text);
+}
+
+.header-form input:focus {
+    outline: none;
+}
+
+.header-form button {
+    padding: 0 1.25rem;
+    color: var(--color-text);
+    background: transparent;
+    border: none; 
+}
+
+.header-avatar {
+    border-radius: 50%;
+    height: 40px;
+    width: 40px;
+}
+
+@media screen and (max-width: 600px) {
+    .container-header {
+        justify-content: space-evenly;
+        padding: 1rem 1rem;
+    }
+    .header-hamburger {
+        display: block;
+    }
+
+    .header-form {
+        flex: 1;
+        margin: 0 1rem;
+    }
+
+    .header-brand {
+        display: none;
+    }
+}
+
+/* SIDEBAR */
+.sidebar {
+    background-color: var(--color-secondary-black);
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    height: 90vh;
+    padding-top: 1rem;
+}
+
+.sidebar .nav-active {
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 1.5rem;
+    margin: 0.2rem 0;
+    cursor: pointer;
+    color: var(--color-text);
+    text-decoration: none;
+}
+
+.nav-active span {
+    margin-left: 0.5rem;   
+    font-size: 0.9em; 
+    cursor: pointer;
+    color: var(--color-text);
+    text-decoration: none;
+}
+
+.sidebar li {
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 1.5rem;
+    margin: 0.2rem 0;
+    cursor: pointer;
+}
+
+.sidebar li>span {
```

Comment:
this has high specificity. Use class on the `span` instead

Reply:
okay, will do that!